src/tls13.c

```diff
@@ -1042,6 +1042,71 @@ int DeriveMasterSecret(WOLFSSL* ssl)
 #define RESUMPTION_LABEL_SZ         10
 /* Resumption label for generating PSK associated with the ticket. */
 static const byte resumptionLabel[RESUMPTION_LABEL_SZ+1] = "resumption";
+
+static int HKDFPSK(byte* okm, word32 okmLen, const byte* prk, word32 prkLen,
```

Comment:
Can we use a better name than `HKDFPSK` like `DerivePSKMasterSecret`?

Reply:
`DeriveResumptionPSKExpand` to be uniform with the caller (`DeriveResumptionPSK`)